test/persistence-hooks.suite.js

```diff
@@ -41,12 +42,19 @@ module.exports = function(dataSource, should, connectorCapabilities) {
         extra: {type: String, required: false},
       });
 
+      GeoModel = dataSource.createModel('GeoModel', {
+        id: {type: String, id: true},
+        name: {type: String, required: false},
+        location: {type: GeoPoint, required: false},
+      });
+
       uid.reset();
 
       if (migrated) {
-        TestModel.deleteAll(done);
+        TestModel.deleteAll();
```

Comment:
You should wait for `TestModel.deleteAll` to complete too. E.g. using `async.series`.